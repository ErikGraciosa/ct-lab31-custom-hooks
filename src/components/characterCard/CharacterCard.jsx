import React from 'react';

function CharacterCard() {
  return (
    <div>Rick Sanchez
    </div>
  );
}

export default CharacterCard;
