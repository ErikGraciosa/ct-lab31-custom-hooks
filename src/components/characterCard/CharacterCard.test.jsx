import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterCard from './CharacterCard';

describe('CharacterCard component', () => {
  afterEach(() => cleanup());
  it('renders CharacterCard', () => {
    const { asFragment } = render(<CharacterCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
