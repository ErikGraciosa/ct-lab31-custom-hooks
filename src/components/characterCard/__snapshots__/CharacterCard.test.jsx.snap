// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterCard component renders CharacterCard 1`] = `
<DocumentFragment>
  <a
    href="characters/undefined"
  >
    <p />
    <img />
  </a>
</DocumentFragment>
`;
